Creación de validadores y pruebas unitarias -feature-Contactos

Endercarias opened the pull request:
Creación de validadores y pruebas unitarias de contactos, cambios en los controladores y las rutas

Endercarias committed:
Creación de validadores y pruebas unitarias

Creación de validadores y pruebas unitarias de contactos, cambios en los controladores y las rutas

Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

Endercarias committed:
Pruebas Unitarias

Se añadieron las pruebas unitarias

Co-Authored-By: Luis Juárez <49957257+LuisJuarez17@users.noreply.github.com>
Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

HJOC15 commented on models/contactos/contactos.js:
```diff
@@ -4,30 +4,36 @@ const { DataTypes } = require("sequelize");
+      allowNull: false,
+    },
+    phone: {
+        type: DataTypes.NUMBER,
```
Si las demás van a estar en español, que teléfono también lo esté.

HJOC15 replied:
Al hacer revisión de los otros códigos que estén todas en inglés.

HJOC15 reviewed (changes requested):
Eliminar la primera mayúscula de ContactModel en todos lados.
Revisar las líneas de código comentadas.

Endercarias committed:
Correcciones y traducción a inglés

Cambios de las propiedades de los modelos de español a inglés y cambio de nombre de contactModel

Co-Authored-By: Luis Juárez <49957257+LuisJuarez17@users.noreply.github.com>
Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

DJPORTILLO30 reviewed (changes requested):
A la espera de confirmación de unit test.

Endercarias committed:
Creación de query

Creación del query de la tabla de contactos

Co-Authored-By: Luis Juárez <49957257+LuisJuarez17@users.noreply.github.com>
Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

DJPORTILLO30 reviewed (approved):
Todo okay

DJPORTILLO30 committed:
Merge branch 'test-branch' into feature-Contactos

HJOC15 reviewed (approved):
¡Todo bien, gracias las correcciones correspondientes!

SFMZ21 reviewed (approved):
Cambios ok

SFMZ21 committed:
Merge branch 'test-branch' into feature-Contactos
